contribute/documentation/README.md

```diff
@@ -1,7 +1,14 @@
 # Contribute to our documentation
 
-Welcome. We're glad you're here to help make our technical documentation even better.
+Welcome! We're glad you're here.
 
-For our style guide and writing guidelines, see [Writers' Toolkit](https://grafana.com/docs/writers-toolkit/).
+[Writers' Toolkit](https://grafana.com/docs/writers-toolkit/) is an open-source project intended to empower both internal and external contributors to help make our technical documentation even better.  
 
-If you're interested in contributing to the Writers' Toolkit, refer to the [Writers' Toolkit](https://github.com/grafana/writers-toolkit) repository.
+Go to [Writers' Toolkit](https://grafana.com/docs/writers-toolkit/) for the following resources:
+
+- **Contribute** - Learn how can you contribute to Grafana Labs documentation.
```

Comment:
This reads a bit funny to me. I think you are using the title of the page but in my head it sounds better as "contribution resources" rather than "contribute resources" but I'm not sure how to restructure the whole list around that change.

Reply:
fixed